apps/docs/components/react/browser-support.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: DefaultLayout
+hideBreadcrumbs: true
+---
+
+# Browser support
+
+This documentation provides information about the browser support for Storefront-ui, a powerful and versatile front-end UI framework. Understanding the browser compatibility of a UI framework is crucial for developers to ensure that their applications work seamlessly across different browsers and versions. This document will outline the supported browsers, their versions, and any specific considerations or known issues related to compatibility.
```

Comment:
```suggestion
Understanding the browser compatibility of a UI framework is crucial for developers to ensure that their applications work seamlessly across different browsers and versions. This document will outline the supported browsers, their versions, and any specific considerations or known issues related to compatibility.
```

I think the first sentence is unnecessary - we can assume that someone knows what Storefront UI is if they're on our docs, and I think just getting right into the document is better than describing it.

Reply:
I agree, thanks https://github.com/vuestorefront/storefront-ui/pull/2883/commits/d9f815bdedbf9008a218a932bd4aa3c303c5f653